tests/test_xist_calendar.py

```diff
@@ -0,0 +1,125 @@
+from unittest import TestCase
+
+import pandas as pd
+from pytz import UTC
+
+from trading_calendars.exchange_calendar_xist import XISTExchangeCalendar
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+
+
+class XISTCalendarTestCase(ExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'xist'
+    calendar_class = XISTExchangeCalendar
+
+    # The XIST is open from 10:00 am to 6:00 pm
+    MAX_SESSION_HOURS = 8.0
+
+    # Daylight savings stopped being observed in 2017 in Turkey
+    DAYLIGHT_SAVINGS_DATES = []
```

Comment:
In `test_trading_calendar.py` there's a `NoDSTExchangeCalendarTestBase` class, should we use that as the base class for this test suite?

Reply:
Oh I didn't even realize we have this.  We could use that class, although looking at the daylight savings test case, making `DAYLIGHT_SAVINGS_DATES = []` bypasses the test as well.  However, it's probably cleaner to use the `NoDSTExchangeCalendarTestBase` so I'm going to make that change.